Winnie can't build main, 4.0.1, and develop branch

On winnie, during compile, get this error:

```
[ 57%] [32mBuilding CXX object src/cpp_common/CMakeFiles/cpp_common.dir/report_messages.cpp.obj[0m
In file included from E:/jenkins/postgresql/rel/pg18w64/include/server/port/win32_port.h:75,
                 from E:/jenkins/postgresql/rel/pg18w64/include/server/port.h:24,
                 from E:/jenkins/postgresql/rel/pg18w64/include/server/c.h:1343,
                 from E:/jenkins/postgresql/rel/pg18w64/include/server/postgres.h:48,
                 from E:/jenkins/pgrouting/branches/main/include/c_common/postgres_connection.h:35,
                 from E:/jenkins/pgrouting/branches/main/src/cpp_common/report_messages.cpp:29:
C:/ming64/mingw64/include/sys/stat.h: In function 'int microsoft_native_fstat(int, microsoft_native_stat*)':
C:/ming64/mingw64/include/sys/stat.h:226:20: error: invalid application of 'sizeof' to incomplete type 'microsoft_native_stat'
  226 |     memset(_Stat,0,sizeof(struct stat));
      |                    ^~~~~~~~~~~~~~~~~~~
C:/ming64/mingw64/include/sys/stat.h:231:8: error: invalid use of incomplete type 'struct microsoft_native_stat'
  231 |   _Stat->st_dev=st.st_dev;
      |        ^~
C:/ming64/mingw64/include/sys/stat.h:174:36: note: forward declaration of 'struct microsoft_native_stat'
  174 | int __cdecl fstat(int _Desc,struct stat *_Stat);
      |                                    ^~~~
C:/ming64/mingw64/include/sys/stat.h:232:8: error: invalid use of incomplete type 'struct microsoft_native_stat'
  232 |   _Stat->st_ino=st.st_ino;
      |        ^~
C:/ming64/mingw64/include/sys/stat.h:174:36: note: forward declaration of 'struct microsoft_native_stat'
  174 | int __cdecl fstat(int _Desc,struct stat *_Stat);
      |                                    ^~~~
C:/ming64/mingw64/include/sys/stat.h:233:8: error: invalid use of incomplete type 'struct microsoft_native_stat'
  233 |   _Stat->st_mode=st.st_mode;
      |        ^~
C:/ming64/mingw64/include/sys/stat.h:174:36: note: forward declaration of 'struct microsoft_native_stat'
  174 | int __cdecl fstat(int _Desc,struct stat *_Stat);
      |                                    ^~~~
C:/ming64/mingw64/include/sys/stat.h:234:8: error: invalid use of incomplete type 'struct microsoft_native_stat'
  234 |   _Stat->st_nlink=st.st_nlink;
      |        ^~
C:/ming64/mingw64/include/sys/stat.h:174:36: note: forward declaration of 'struct microsoft_native_stat'
  174 | int __cdecl fstat(int _Desc,struct stat *_Stat);
      |                                    ^~~~
C:/ming64/mingw64/include/sys/stat.h:235:8: error: invalid use of incomplete type 'struct microsoft_native_stat'
  235 |   _Stat->st_uid=st.st_uid;
      |        ^~
C:/ming64/mingw64/include/sys/stat.h:174:36: note: forward declaration of 'struct microsoft_native_stat'
  174 | int __cdecl fstat(int _Desc,struct stat *_Stat);
      |                                    ^~~~
C:/ming64/mingw64/include/sys/stat.h:236:8: error: invalid use of incomplete type 'struct microsoft_native_stat'
  236 |   _Stat->st_gid=st.st_gid;
      |        ^~
C:/ming64/mingw64/include/sys/stat.h:174:36: note: forward declaration of 'struct microsoft_native_stat'
  174 | int __cdecl fstat(int _Desc,struct stat *_Stat);
      |                                    ^~~~
C:/ming64/mingw64/include/sys/stat.h:237:8: error: invalid use of incomplete type 'struct microsoft_native_stat'
  237 |   _Stat->st_rdev=st.st_rdev;
      |        ^~
C:/ming64/mingw64/include/sys/stat.h:174:36: note: forward declaration of 'struct microsoft_native_stat'
  174 | int __cdecl fstat(int _Desc,struct stat *_Stat);
      |                                    ^~~~
C:/ming64/mingw64/include/sys/stat.h:238:8: error: invalid use of incomplete type 'struct microsoft_native_stat'
  238 |   _Stat->st_size=(_off_t) st.st_size;
      |        ^~
C:/ming64/mingw64/include/sys/stat.h:174:36: note: forward declaration of 'struct microsoft_native_stat'
  174 | int __cdecl fstat(int _Desc,struct stat *_Stat);
      |                                    ^~~~
C:/ming64/mingw64/include/sys/stat.h:239:8: error: invalid use of incomplete type 'struct microsoft_native_stat'
  239 |   _Stat->st_atime=st.st_atime;
      |        ^~
C:/ming64/mingw64/include/sys/stat.h:174:36: note: forward declaration of 'struct microsoft_native_stat'
  174 | int __cdecl fstat(int _Desc,struct stat *_Stat);
      |                                    ^~~~
C:/ming64/mingw64/include/sys/stat.h:240:8: error: invalid use of incomplete type 'struct microsoft_native_stat'
  240 |   _Stat->st_mtime=st.st_mtime;
      |        ^~
C:/ming64/mingw64/include/sys/stat.h:174:36: note: forward declaration of 'struct microsoft_native_stat'
  174 | int __cdecl fstat(int _Desc,struct stat *_Stat);
      |                                    ^~~~
C:/ming64/mingw64/include/sys/stat.h:241:8: error: invalid use of incomplete type 'struct microsoft_native_stat'
  241 |   _Stat->st_ctime=st.st_ctime;
      |        ^~
C:/ming64/mingw64/include/sys/stat.h:174:36: note: forward declaration of 'struct microsoft_native_stat'
  174 | int __cdecl fstat(int _Desc,struct stat *_Stat);
      |                                    ^~~~
E:/jenkins/postgresql/rel/pg18w64/include/server/port/win32_port.h: At global scope:
E:/jenkins/postgresql/rel/pg18w64/include/server/port/win32_port.h:254:8: error: redefinition of 'struct stat'
  254 | struct stat                                             /* This should match struct __stat64 */
      |        ^~~~
In file included from C:/ming64/mingw64/include/wchar.h:413,
                 from C:/ming64/mingw64/include/c++/13.2.0/cwchar:44,
                 from C:/ming64/mingw64/include/c++/13.2.0/bits/postypes.h:40,
                 from C:/ming64/mingw64/include/c++/13.2.0/iosfwd:42,
                 from C:/ming64/mingw64/include/c++/13.2.0/ios:40,
                 from C:/ming64/mingw64/include/c++/13.2.0/istream:40,
                 from C:/ming64/mingw64/include/c++/13.2.0/sstream:40,
                 from E:/jenkins/pgrouting/branches/main/include/cpp_common/report_messages.hpp:30,
                 from E:/jenkins/pgrouting/branches/main/src/cpp_common/report_messages.cpp:26:
C:/ming64/mingw64/include/_mingw_stat64.h:40:10: note: previous definition of 'struct stat'
   40 |   struct stat {
      |          ^~~~
make[2]: *** [src/cpp_common/CMakeFiles/cpp_common.dir/build.make:160: src/cpp_common/CMakeFiles/cpp_common.dir/report_messages.cpp.obj] Error 1
make[1]: *** [CMakeFiles/Makefile2:1887: src/cpp_common/CMakeFiles/cpp_common.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

We had this issue on 4.0 before and was fixed for 4.0.0 release. But this looks to be a different file.  Dev branch is also failing

--- a/src/cpp_common/report_messages.cpp
+++ b/src/cpp_common/report_messages.cpp
@@ -23,14 +23,14 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
  ********************************************************************PGR-GNU*/
 
-#include "cpp_common/report_messages.hpp"
-
 extern "C" {
 #include "c_common/postgres_connection.h"
 #include "c_common/e_report.h"
 }
 
 #include <sstream>
+
+#include "cpp_common/report_messages.hpp"
 
 #include "cpp_common/alloc.hpp"
 

